www/front_src/src/Resources/Filter/index.tsx

```diff
@@ -142,7 +142,7 @@ const Filter = ({
 }: Props): JSX.Element => {
   const classes = useStyles();
 
-  const [searchFieldValue, setSearchFieldValue] = useState<string>();
+  const [searchFieldValue, setSearchFieldValue] = React.useState<string>();
```

Comment:
Why it's necessary to specify "React" here and not on "import" ?

Reply:
Because the recommanded import syntax is : `import * as React from 'react;`